clients/license.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/gofiber/fiber/v2"
 )
 
+type GetReplicatedCustomerResponse struct {
+	ReplicatedEmail string `json:"replicatedEmail"`
+	Message         string `json:"message"`
+	StatusCode      string `json:status_code"`
```

Comment:
Added missed quote. The case difference is because that's how it is designed in replicated. The json field naming is not done by us. We are just consuming the json.

Reply:
Got it. Resolving the comment